app/src/index.ts

```diff
@@ -78,6 +83,18 @@ const configuration: ConfigurationContext = {
   useCustomNotifications: useNotifications,
   useAttestation: useAttestation,
   enableUseMultUseInvitation: false,
+  enablePushNotifications: {
+    status: status,
+    setup: setup,
+    toggle: async (state: boolean, agent: Agent<any>) => {
+      if (state) {
+        await activate(agent)
+      } else {
+        await deactivate(agent)
+      }
+    },
+    check: async () => 'granted',
```

Comment:
Should this be `NotificationPermissionStatus.GRANTED` (or `.Granted`)?

Reply:
actually that's an artifact from an old change, I'll remove it